typescript/sdk/src/router/types.ts

```diff
@@ -56,10 +57,11 @@ export enum RouterViolationType {
 
 export interface RouterViolation extends CheckerViolation {
   type: RouterViolationType.EnrolledRouter;
-  remoteChain: string;
   contract: Router;
-  actual: string;
-  expected: string;
+  routerDiff: ChainMap<{
+    actual: AddressBytes32;
+    expected: AddressBytes32;
+  }>;
```

Comment:
This is fine but would it be better to decode the address from bytes to string to make it easier for consumers of these violations to work with?

Reply:
I think it's better to leave as-is here, so the governor (which works with this) doesn't have to do any translation from the values that its respective checker provides